### PR TITLE
Add possibility to write text with a markdown "about" file on the title page

### PR DESCRIPTION
**Pull request by davidkroell**
This adds the option (behind a feature-flag) to write more text onto the home page.

I had to change the source branch and the other PR #170 was unfortunately automatically closed.

Superceeds #170

### CODE REVIEW

**Comment by davidkroell** [comment]
@yashmehrotra I had to create another PR, which is essentially the same as #170.

> Do we need this 3rd type ? Since its only for the homepage, can we just infer it via filename and its existense ?

Yea probably, but I'm fairly new to Hugo, so I don't know how. If you can provide a path for me, I can implement it.

**Comment by yashmehrotra** [comment]
> Yea probably, but I'm fairly new to Hugo, so I don't know how. If you can provide a path for me, I can implement it.
Since the only place to add about is the index page, the path of the about file can be static. (For eg., content/index-about.md)

Our logic then will be:
- Check if that file exists
- If yes, include partial else ignore

File: exampleSite/config.toml [hunk 1]
```diff
@@ -12,6 +12,7 @@ pygmentsStyle = "monokai"
 [params]
   footer = "The Marauders"
   description = "Sky above, sand below and peace within"
+  useMDAbout = true # set this to true to use the extended home page description with markdown
```
Comment (yashmehrotra) [inline exampleSite/config.toml:15]: When we are inferring from the path, this is also not needed

**Comment by davidkroell** [comment]
@yashmehrotra I made the necessary changes, and the new version is far more straightforward.

**Comment by yashmehrotra** [comment]
Thanks for the PR @davidkroell

**Pull request merged**